release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -115,6 +115,116 @@ The web console is now localized and provides language support for global users.
+* Multipathing
+* {product-title} Pipelines TP
+* {product-title} Service Mesh
+* OVN-Kubernetes with an initial installation of {product-title} 4.7
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* z/VM Emulated FBA devices on SCSI disks
+
+[discrete]
+===== Supported features
+
+The following features are also supported on IBM Z and LinuxONE:
+
+* CodeReady Workspaces
+* {product-title} Do (odo)
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+
+[discrete]
+===== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Log forwarding
+** Precision Time Protocol (PTP) hardware
+** CSI volume snapshots
+
+* The following {product-title} features are unsupported:
+** {product-title} Virtualization
+** CodeReady Containers (CRC)
+** {product-title} Metering
+** Multus CNI plug-in
+** FIPS cryptography
+** Encrypting data stored in etcd
+** Automatic repair of damaged machines with machine health checking
+** Tang mode disk encryption during {product-title} deployment
+** {product-title} Serverless
+** Helm command-line interface (CLI) tool
+** Controlling overcommit and managing container density on nodes
+** etcd cluster Operator
+** CSI volume cloning
+** NVMe
+** 4K FCP block device
+
+* Worker nodes must run {op-system-first}.
+* Persistent shared storage must be provisioned by using either NFS or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA.
+* These features are available only for {product-title} on IBM Z for {product-version}:
+** HyperPAV enabled on IBM System Z /LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[id="ocp-4-7-ibm-power"]
+==== IBM Power Systems
+
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power Systems] or xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power Systems in a restricted network] for installation instructions.
+
+[discrete]
+===== Notable Enhancements
+
+The following new features are supported on IBM Power Systems with {product-title} {product-version}:
+
+* Multipathing
+* {product-title} Pipelines TP
```

Comment:
@vikram-redhat, why don't we add the product names for the aligned products to the main attributes file again?

Reply:
Pipelines is technically not an aligned product but a feature of OCP. Are you asking if we should have `{piplines-title}` (or similar) instead of `{product-title} Pipelines TP`?